src/components/CustomDisciplineItem.tsx

```diff
@@ -82,9 +73,9 @@ const CustomDisciplineItem = ({
         </DisciplineItem>
       ) : (
         <Placeholder>
-          <ErrorText>
+          <ContentSecondaryLight>
```

Comment:
```js
const ErrorText = styled.Text`
  font-size: ${props => props.theme.fonts.defaultFontSize};
  font-weight: ${props => props.theme.fonts.lightFontWeight};
  font-family: ${props => props.theme.fonts.contentFontRegular};
  color: ${props => props.theme.colors.primary};
  `
```

But this is misleading because it is was not color `incorrect` before so it does not differ in style to a normal `ContentSecondaryLight` so either the text should change to `incorrect` or we should keep it like that

Reply:
you are right, sorry, thought it was red before.